HeterogeneousCore/CUDAUtilities/interface/cudastdAlgorithm.h

```diff
@@ -8,76 +8,63 @@
 // reimplementation of std algorithms able to compile with CUDA and run on GPUs,
 // mostly by declaringthem constexpr
 
-namespace cuda_std  {
+namespace cuda_std {
 
-  template<typename T = void>
+  template <typename T = void>
   struct less {
-    __host__ __device__
-    constexpr bool operator()(const T &lhs, const T &rhs) const {
-      return lhs < rhs;
-    }
+    __host__ __device__ constexpr bool operator()(const T &lhs, const T &rhs) const { return lhs < rhs; }
```

Comment:
as for `__global__`, I would really prefer to keep `__host__` and `__device__` attributes  on a separate line